src/docfx/lib/git/GitUtility.cs

```diff
@@ -155,7 +155,7 @@ public static void Fetch(SecretConfig secrets, string path, string url, string r
 
         var (http, secret) = GetGitCommandLineConfig(url, secrets);
 
-        ExecuteNonQuery(path, $"{http} -c core.longpaths=true fetch --progress {options} \"{url}\" {refspecs}", secret);
+        ExecuteNonQuery(path, $"{http} -c credential.helper=\"\" -c core.longpaths=true fetch --progress {options} \"{url}\" {refspecs}", secret);
```

Comment:
Set environment variable `GIT_TERMINAL_PROMPT` to `0` in azure pipeline yml for running tests only. Changing the source code here breaks experiences that expects user interaction.

Reply:
The OP build needs disable the prompt, otherwise, build hangs for invalid git access token.

I had tested `$env:GIT_TERMINAL_PROMPT=0`, but it didn't look like work for me, I haven't found a working environment variable for now, otherwise, we may not need change in docfx.

But we can try if installing git without git credential manager can work or not

Reply:
OP already disabled it here: https://dev.azure.com/ceapex/Engineering/_git/OpenPublishing.Build?path=/Applications/LocalRun/WorkerRole.cs&version=GBdevelop&line=54&lineEnd=54&lineStartColumn=49&lineEndColumn=68&lineStyle=plain&_a=contents

Reply:
Let me confirm with Peng Cui first

Reply:
@yufeih @herohua Peng is OOF, but the `GIT_TERMINAL_PROMPT` seems not work. The [bug](https://dev.azure.com/ceapex/Engineering/_workitems/edit/376820/) is created on Feb 2021, however, the [`GIT_TERMINAL_PROMPT=0` had been set in OPS on Mar 2020(even early)](https://dev.azure.com/ceapex/Engineering/_git/OpenPublishing.Build/commit/8f67cc5fe41d3d70ea0c46b1925826db78b66266?refName=refs/heads/develop&path=/Applications/LocalRun/WorkerRole.cs&_a=contents).

Should we handle this in Docfx side? What I think is we can try to handle this when installing git, otherwise, we can have a switch to control if we want to disable the prompt in docfx.